e2e/app/component/select-menu.ts

```diff
@@ -87,11 +86,15 @@ export default class SelectMenu {
   private async isOpen() {
     const selector = this.dropdownXpath() +
        '/*[contains(concat(" ", normalize-space(@class), " "), " p-dropdown-panel ")]';
-    const panel = await this.page.waitForXPath(selector);
-    const classNameString = await (await panel.getProperty('className')).jsonValue();
-    const splits = classNameString.toString().split(' ');
-    await panel.dispose();
-    return splits.includes('p-input-overlay-visible');
+    try {
+      const panel = await this.page.waitForXPath(selector);
+      const classNameString = await (await panel.getProperty('className')).jsonValue();
+      const splits = classNameString.toString().split(' ');
+      await panel.dispose();
+      return splits.includes('p-input-overlay-visible');
+    } catch (err) {
+      return false;
```

Comment:
Should we also `console.error` this, or is it expected?

Reply:
handle both expected or unexpected.